### DISCUSSION TITLE
Local development starts v2.0.0

### DISCUSSION THREAD

**Discussion by chris-smearing** [requarks/wiki#3022] [Help / Questions] [answered]
I've read the development docs carefully and am trying to install on Windows WSL for local development (to change the theme). I have done the following:

- cloned the project
- edited config.yml
- run yarn
- run yarn dev

When I open localhost:3000, the page says it is running v2.0.0. I confirmed the git branch is 'dev', but also tried branch '2.5.170'. Either way, the page says I'm installing v2.0.0.

![image](https://user-images.githubusercontent.com/2591639/106923474-b8d48480-66d3-11eb-83ad-a38bcad50507.png)

Does anyone know what I'm doing wrong?

**Comment by NGPixel** [comment, marked as answer]
The dev code is always 2.0.0. The actual version is injected during the build, as detailed [here](https://docs.requarks.io/dev/build-process).

**Reply by chris-smearing** [reply]
Thank you! I'm new to the project and was unsure what was happening.
